Converter/Calculator/Program.cs

```diff
@@ -0,0 +1,101 @@
+﻿    using System.Threading.Channels;
+    using Calculator;
+
+    bool c;
+    int b;
+    do
+    {
+        Console.Write("\n Write a number to convert: ");
```

Comment:
There is no need for separate number input here. It would be better to keep the whole input process in one place